src/commcare_cloud/ansible/roles/commcarehq/templates/celery_task_kill.sh.j2

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash 
+
+# Process to soft and hard kill old celery jobs.
+#
+
+HOURS=$1
+MAX_MINUTES=`expr ${HOURS} \* 60 \* 60`
+
+SIG_NAME=$2
+
+PID=$(ps -eo comm,pid,etimes | awk '/celeryd:/ { if ( $4 > '"${MAX_MINUTES}"' ) { print $3 } }');
+
+if [ "$PID" != "" ];
+then
+	kill --verbose -"${SIG_NAME}" $(ps -eo comm,pid,etimes | awk '/celeryd:/ { if ( $4 > '"${MAX_MINUTES}"' ) { print $3 } }');
```

Comment:
I don't think `kill` has a verbose option.

Perhaps you could do something like:
1. get list of program names + PIDs to kill
2. convert that to list of PIDs
3. Echo list of program names + PIDs
4. issue kill command
5. Check that all PIDs are terminated (? maybe)

Reply:
OK interesting. The ubuntu man page doesn't indicate a verbose option: https://manpages.ubuntu.com/manpages/bionic/man1/kill.1.html

When you tested it were the processes that matched? Perhpas it didn't execute the kill command.

Reply:
yes, the --verbose could have failed and I should have tested/caught it, I moved forward with it as it was mentioned in man-pages document. Going forward I will cross check twice. I removed it from the file now. thanks Simon.